Tests/SerializationTests.swift

```diff
@@ -17,5 +17,45 @@ class SerializationTests: QuickSpec {
                 expect(encodedString).to(equal(#"{"id":"TheUserID","type":1}"#))
             }
         }
+
+        describe("Campaign model") {
+            context("When parsing title for contexts") {
+
+                it("will return empty array if there are no contexts") {
+                    let campaign = TestHelpers.generateCampaign(id: "id", title: "title")
+                    expect(campaign.contexts).to(beEmpty())
+                }
+
+                it("will properly read context if there's nothing beside it") {
```

Comment:
"will properly read context when it is the only string content"

Reply:
Much better 👍